clients/client-java/src/main/java/com/datastrato/gravitino/client/GravitinoFilesetCatalog.java

```diff
@@ -210,8 +214,23 @@ static String formatFilesetRequestPath(Namespace ns) {
         .toString();
   }
 
+  /** Encode the identifier to be used in the URL. */
+  static String encoder(String ident) {
```

Comment:
The method `com.datastrato.gravitino.rest.RESTUtils#encodeString` can be reused. 

BTW, can you also help to replace the method `URLEncoder.encode` in `com.datastrato.gravitino.client.RelationalTable#formatPartitionRequestPath`

Reply:
done